invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImageGrid.tsx

```diff
@@ -36,7 +37,7 @@ const GalleryImageGrid = () => {
     );
   }
 
-  if (isLoading) {
+  if (isLoading || isFetching) {
```

Comment:
This introduces a flash of the no-content fallback while pages load. Can we represent the fetching state in some other way? Perhaps a little spinner replacing the clear icon on the search input?